modulesets-stable/gtk-osx-gstreamer.modules

```diff
@@ -2,15 +2,13 @@
 <!DOCTYPE moduleset SYSTEM "moduleset.dtd">
 <?xml-stylesheet type="text/xsl" href="moduleset.xsl"?>
 <moduleset>
-  <repository type="tarball" name="ftp.gnome.org" default="yes"
-              href="http://ftp.gnome.org/pub/GNOME/sources/"/>
   <repository type="tarball" name="gstreamer"
```

Comment:
This won't work without specifying the  repo in each module unless you add ```default="yes"``` here.